From: =?UTF-8?q?Rafa=C5=82=20Mi=C5=82ecki?= <zajec5@gmail.com>
Date: Mon, 7 Dec 2015 20:51:32 +0100
Subject: [PATCH] ARM: BCM: Fix SMP in Kconfig entries CYGNUS and 5301X
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Patch "ARM: BCM: Add SMP support for Broadcom 4708" added symbol in a
wrong place.

Signed-off-by: Rafał Miłecki <zajec5@gmail.com>
---
 arch/arm/mach-bcm/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/mach-bcm/Kconfig
+++ b/arch/arm/mach-bcm/Kconfig
@@ -29,7 +29,6 @@ config ARCH_BCM_IPROC
 config ARCH_BCM_CYGNUS
 	bool "Broadcom Cygnus Support" if ARCH_MULTI_V7
 	select ARCH_BCM_IPROC
-	select HAVE_SMP
 	help
 	  Enable support for the Cygnus family,
 	  which includes the following variants:
@@ -58,6 +57,7 @@ config ARCH_BCM_5301X
 	select ARM_ERRATA_754322
 	select ARM_ERRATA_775420
 	select ARM_ERRATA_764369 if SMP
+	select HAVE_SMP
 
 	help
 	  Support for Broadcom BCM470X and BCM5301X SoCs with ARM CPU cores.
